### PR TITLE
added product list wishlist and filters

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review it urgently.

### CODE REVIEW

File: CartMgmt/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  width: 2rem;
+  border: 1px solid var(--grey-color);
+  border-radius: 50%;
+}
+.container-flex{
+margin-bottom: auto;
+border-color: none;
+}
+
+hr.someClass {
```
Comment (sharmivijay) [inline CartMgmt/Cart/cart.css]: use class name that are self descriptive

File: CartMgmt/Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
```
Comment (sharmivijay) [inline CartMgmt/Cart/cart.css]: you can move this variables file

File: CartMgmt/Cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+.container-flex{
+margin-bottom: auto;
+border-color: none;
+}
+
+hr.someClass {
+  background-color: black;
+  width: 50%;
+  height: 2px;
+  border-top: 1px solid black;
```
Comment (sharmivijay) [inline CartMgmt/Cart/cart.css]: use hexacodes for the colors and have it in variables

File: ProductListing/ProductListing/Wishlist/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
```
Comment (sharmivijay) [inline ProductListing/ProductListing/Wishlist/wishlist.css:8]: same here

File: ProductListing/ProductListing/product.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root{
```
Comment (sharmivijay) [inline ProductListing/ProductListing/product.css:8]: same here

**Review by sharmivijay — APPROVED** [review]
Add few minor changes. Code looks great

File: CartMgmt/Cart/cart.css (hunk 1 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css]: yes will do

File: CartMgmt/Cart/cart.css (hunk 2 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css]: ok

File: CartMgmt/Cart/cart.css (hunk 3 above)
Comment (MonikaJhingan) [reply CartMgmt/Cart/cart.css]: ok

File: ProductListing/ProductListing/Wishlist/wishlist.css (hunk 4 above)
Comment (MonikaJhingan) [reply ProductListing/ProductListing/Wishlist/wishlist.css:8]: ok

File: ProductListing/ProductListing/product.css (hunk 5 above)
Comment (MonikaJhingan) [reply ProductListing/ProductListing/product.css:8]: ok

**Review by aamrapaliwandhre — APPROVED** [review]

File: ProductListing/ProductListing/Wishlist/wishlist.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  font-size: 3rem;
+}
+
+.wishlist-item {
+  flex: 50;
+  margin: 1rem;
+}
+
+.icon-color {
+  color: #ff0000;
```
Comment (atharvabhanage02) [inline ProductListing/ProductListing/Wishlist/wishlist.css:29]: Use color variables here,

File: ProductListing/ProductListing/Wishlist/wishlist.css [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+}
+.icon-color:hover {
+  color: #dd2626;
```
Comment (atharvabhanage02) [inline ProductListing/ProductListing/Wishlist/wishlist.css:32]: same here as well

**Review by atharvabhanage02 — APPROVED** [review]
Few Suggestions , else everything looks fine

**Review by neetukumari4858 — APPROVED** [review]
code is readable its good to merge

File: ProductListing/ProductListing/Wishlist/wishlist.css (hunk 6 above)
Comment (MonikaJhingan) [reply ProductListing/ProductListing/Wishlist/wishlist.css:29]: ok

File: ProductListing/ProductListing/Wishlist/wishlist.css (hunk 7 above)
Comment (MonikaJhingan) [reply ProductListing/ProductListing/Wishlist/wishlist.css:32]: ok

**Pull request merged**